roles/ravendb_node/molecule/plugins-unsecured-nodes/molecule.yml

```diff
@@ -0,0 +1,64 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+  port_bindings:
+    '8080': 80
+    '38888': 38888
+
+platforms:
+  - name: ubuntu-bionic-node-a
```

Comment:
Ubuntu Bionic is out of support, I believe we can remove it within the next iteration?